.github/workflows/lint.yml

```diff
@@ -133,30 +133,69 @@ jobs:
           channel: beta
       - id: checkout
         uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28
-      - id: packages_auth_amplify_auth_cognito_pub_upgrade
-        name: packages/auth/amplify_auth_cognito; flutter pub pub upgrade
+      - id: packages_analytics_amplify_analytics_pinpoint_example_ios_.symlinks_plugins_amplify_secure_storage_pub_upgrade
+        name: packages/analytics/amplify_analytics_pinpoint/example/ios/.symlinks/plugins/amplify_secure_storage; flutter pub pub upgrade
         if: "always() && steps.checkout.conclusion == 'success'"
-        working-directory: packages/auth/amplify_auth_cognito
+        working-directory: packages/analytics/amplify_analytics_pinpoint/example/ios/.symlinks/plugins/amplify_secure_storage
         run: flutter pub pub upgrade
-      - name: "packages/auth/amplify_auth_cognito; dart format --output=none --set-exit-if-changed ."
-        if: "always() && steps.packages_auth_amplify_auth_cognito_pub_upgrade.conclusion == 'success'"
-        working-directory: packages/auth/amplify_auth_cognito
+      - name: "packages/analytics/amplify_analytics_pinpoint/example/ios/.symlinks/plugins/amplify_secure_storage; dart format --output=none --set-exit-if-changed ."
+        if: "always() && steps.packages_analytics_amplify_analytics_pinpoint_example_ios_.symlinks_plugins_amplify_secure_storage_pub_upgrade.conclusion == 'success'"
+        working-directory: packages/analytics/amplify_analytics_pinpoint/example/ios/.symlinks/plugins/amplify_secure_storage
         run: "dart format --output=none --set-exit-if-changed ."
-      - name: "packages/auth/amplify_auth_cognito; flutter analyze --fatal-infos lib"
-        if: "always() && steps.packages_auth_amplify_auth_cognito_pub_upgrade.conclusion == 'success'"
-        working-directory: packages/auth/amplify_auth_cognito
-        run: flutter analyze --fatal-infos lib
+      - name: "packages/analytics/amplify_analytics_pinpoint/example/ios/.symlinks/plugins/amplify_secure_storage; flutter analyze --fatal-infos lib pigeons"
```

Comment:
Seems weird we're adding all these changes for just adding flutter library?  This seems to be unrelated to the PR, perhaps we could add a comment explaining what additional changes were made here as well.

Reply:
Didn't mean to - it's mono_repo being silly